srcpkgs/wiredtiger/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'wiredtiger'
 pkgname=wiredtiger
 version=2.9.3
```

Comment:
how about updating to the latest release 3.1.0 ?

Reply:
Yes, that makes sense, but i think we should fix the referenced issue with this request first, because currently it breaks MongoDB.